bootstrap/bootstrap.go

```diff
@@ -178,7 +182,13 @@ func (g *Generator) Run() error {
 		defer os.Remove(f.Name()) // will not remove after rename
 	}
 
-	cmd := exec.Command("go", "run", "-tags", g.BuildTags, filepath.Base(path))
+	execArgs := []string{"run"}
+	if g.GenBuildFlags != "" {
+		execArgs = append(execArgs, strings.Split(g.GenBuildFlags, buildArgsSeparator)...)
```

Comment:
But if you have

```
-gen_build_flags "-example 'text with space'"
```

Reply:
Hey @GoWebProd - thanks for quick review!
I've fixed the issue, please take a look.

```
bin/easyjson -all -gen_build_flags="-mod=mod -gccgoflags '-march=native -O3'" ./tests/data.go
Regexped flags [-mod=mod -gccgoflags '-march=native -O3']
Regexped flags len 3
[go run -mod=mod -gccgoflags '-march=native -O3' -tags  easyjson-bootstrap433561849.go]

bin/easyjson -all -gen_build_flags="-mod=mod" ./tests/data.go
Regexped flags [-mod=mod]
Regexped flags len 1
[go run -mod=mod -tags  easyjson-bootstrap252406791.go]

bin/easyjson -all -gen_build_flags="-mod=mod -x" ./tests/data.go
Regexped flags [-mod=mod -x]
Regexped flags len 2
[go run -mod=mod -x -tags  easyjson-bootstrap003937401.go]

bin/easyjson -all -gen_build_flags="" ./tests/data.go
[go run -tags  easyjson-bootstrap846570929.go]
```